src/Orbit.js

```diff
@@ -11,7 +11,7 @@ const OrbitUser    = require('./orbit-user')
 const IdentityProviders = require('./identity-providers')
 
 const logger = Logger.create("Orbit", { color: Logger.Colors.Green })
-require('logplease').setLogLevel('ERROR')
+require('logplease').setLogLevel('DEBUG')
```

Comment:
Note that src/Orbit.js is not used anymore. Orbit code comes from module `orbit_` and is only required in the UI (App.js)